ccm_web/package.json

```diff
@@ -59,7 +59,7 @@
     "sequelize": "^6.9.0",
     "sequelize-typescript": "^2.1.1",
     "swagger-ui-express": "^4.1.6",
-    "umzug": "^3.0.0-beta.16",
+    "umzug": "~3.0.0",
```

Comment:
I see you are allowing only patch version (`~`) and not minor version  (`^`) updates (we're doing the latter in most other cases, I think cause that is the default). https://docs.npmjs.com/about-semantic-versioning#using-semantic-versioning-to-specify-update-types-your-package-can-accept

Do you think we should change all these ranges to `~` before we release?

Reply:
It is a debatable question which way we want to go. I will change it to `^`  and decide on changing it to `~` later

Reply:
I changed it and ran `npm install` so that `package-lock.json` get updated as well